dodo.py

```diff
@@ -143,6 +155,29 @@ def task_rebuild():
             'actions': [],
         }
 
+def task_test():
+    """
+    DoIt Task: Runs smoke tests against branches on GitHub.
+
+    Usage: doit test:<project> 
+    """
+    for project in ALL_PROJECTS:
+        yield {
+            'name': project.name,
+            'actions': [project.test],
+        }
+
+def task_merge():
+    """
+    DoIt Task: Runs smoke tests against branches on GitHub.
```

Comment:
(this is probably _my_ copy paste error but still) The docstring doesn't seem to match what the command does

Reply:
Resolved.